foam_controlwrapper/foam_files.py

```diff
@@ -1,61 +1,91 @@
 """ foam_files module
 
-Module for writing OpenFOAM initial files
+Module for writing and modifying OpenFOAM files
 
 """
 import os
 from foam_templates import head, dictionaryTemplates
 from foam_templates import scalarTemplates, vectorTemplates
+from simphony.core.cuba import CUBA
+from cuba_extension import CUBAExt
+from PyFoam.RunDictionary.ParsedParameterFile import ParsedParameterFile
+from PyFoam.Basics.DataStructures import Vector
 
 
 class FoamFiles():
+    """ Module for OpenFOAM file management
```

Comment:
more information in the `new style`, `old style` issue can be found here http://stackoverflow.com/questions/2715186/why-would-it-be-necessary-to-subclass-from-object-in-python